packages/bvaughn-architecture-demo/components/sources/SearchResultHighlight.tsx

```diff
@@ -1,24 +1,91 @@
+import { ReactNode } from "react";
+
 import styles from "./SearchResultHighlight.module.css";
 
+// Must stay in sync with --column-breakpoint-width in SourceListRow.module.css
+const COLUMN_BREAKPOINT_WIDTH_IN_PIXELS = 20;
```

Comment:
This feels like an unfortunate hack. Maybe I can think of a way to decouple these.